tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "noUnusedLocals": true,
+    "strict": false,
```

Comment:
shouldn't it be true?

Reply:
I'm importing icons and they are implicitly `any` because i didn't write types for them

Reply:
`{ 'noicon':  () => null }` should work (I guess)